src/app/login/login.component.html

```diff
@@ -24,6 +24,9 @@ <h3 class="available-containers">
     </div>
   </div>
 </div>
+<div class="text-center">
```

Comment:
Ideally the page would be converted to use the header component for consistency, but this is fine too.